addons/im_livechat/static/src/public/session.js

```diff
@@ -0,0 +1,7 @@
+odoo.define('web.session', function (require) {
+
+    const Session = require('web.Session');
+    const { serverUrl } = require('im_livechat.loaderData');
+
+    return new Session(undefined, serverUrl, { use_cors: true });
```

Comment:
modules was given as option before diff, no longer used?

Reply:
I tested it and it was empty and unused, so I removed it as code cleaning.